exercises/07.csrf/01.solution.setup/README.mdx

```diff
@@ -12,7 +12,9 @@ IjdDa3p6WkV1d3NNel&nev3r&60nna&gIv3&yoU&uP&UW0zYmxDMDNyRjQi.NM71601wmCvZ%2FZaGIG
 ```
 
 And while you can't visually verify it, that (unsigned) token value is in the
-browser as well. You can check by executing this in the console:
+browser as well. You can check by executing this in the console (note that this
```

Comment:
this is what I ran into when I ran this in console, but let me know if you want me to remove this edit